src/registries/menus/topbar_menu_registry.ts

```diff
@@ -377,6 +377,14 @@ topbarMenuRegistry
     ...ACTION_DATA.sortDescending,
     sequence: 20,
   })
+  .addChild("data_cleanup", ["data"], {
+    ...ACTION_DATA.dataCleanup,
+    sequence: 20,
+  })
+  .addChild("remove_duplicates", ["data", "data_cleanup"], {
+    ...ACTION_DATA.removeDuplicates,
+    sequence: 10,
+  })
   .addChild("split_to_columns", ["data"], {
     ...ACTION_DATA.splitToColumns,
     sequence: 20,
```

Comment:
Shouldn't this sequence be increased to take the new item into account ?

Reply:
I will downgrade the sequence of the new item --> 15